package.json

```diff
@@ -30,7 +30,7 @@
     "is-git-url": "^1.0.0",
     "js-yaml": "^3.13.1",
     "lodash.pick": "^4.4.0",
-    "mesg-js": "^4.3.1",
+    "mesg-js": "github:mesg-foundation/mesg-js#build/add-mapping-constant-support",
```

Comment:
We want rather version of mesg-js here, won't we?

Reply:
yes this is just temporary to test the feature and shouldn't be merged until the dependency https://github.com/mesg-foundation/mesg-js/pull/132 is merged and released as beta. There is a dependency for this PR